pihole

```diff
@@ -293,7 +293,6 @@ Options:
   Try 'pihole logging --help' for more information."
     exit 1
   fi
-  restartDNS
```

Comment:
FTL will restart itself when required after logging has changed?

Reply:
I believe so... paging FTL master @DL6ER for confirmation

Reply:
Yes.
The `dns.queryLogging` config has the flag `FLAG_RESTART_FTL` set:

https://github.com/pi-hole/FTL/blob/6c0c87547653e3bb33340dfa61c06f7b0cb88d7b/src/config/config.c#L560-L565

Reply:
:) Everything has been said